src/dotnet/Fable.Compiler/Transforms/State.fs

```diff
@@ -81,10 +81,9 @@ type Project(projectOptions: FSharpProjectOptions, implFiles: Map<string, FSharp
 
 /// Type with utilities for compiling F# files to JS
 /// No thread-safe, an instance must be created per file
-type Compiler(currentFile, project: Project, options) =
+type Compiler(fableCore, currentFile, project: Project, options) =
     let mutable id = 0
     let logs = Dictionary<string, string list>()
-    let fableCore = (Path.getRelativePath currentFile project.FableCore).TrimEnd('/')
```

Comment:
Again, if we deprecate passing fableCore from the JS client and use only the CLI we don't need these changes.

Reply:
This line was not changed, just moved it from within the constructor to a parameter so we can easily construct a Compiler with specific fableCore path, this way the Compiler is a bit more flexible.